File: TreeViewAdvDemo/C#/Form1.cs
using Syncfusion.Drawing;
using Syncfusion.Windows.Forms.Tools;
using System.Drawing;
using System.Windows.Forms;

namespace TreeViewAdvDemo
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();

            this.treeViewAdv1.SelectionMode = TreeSelectionMode.MultiSelectAll;

            // To ensure that the selected node is highlighted always
            this.treeViewAdv1.HideSelection = false;

            // The appearence of selection rectangle can be changed by following property
            // To identify selected node is highlighted or not when TreeViewAdv loses focus           
            // Set custom colors to the selection rectangle
            this.treeViewAdv1.InactiveSelectedNodeBackground = new BrushInfo(Color.Green);
            this.treeViewAdv1.InactiveSelectedNodeForeColor = Color.White;            
        }
    }
}

File: TreeViewAdvDemo/VB/Form1.vb
Imports Syncfusion.Drawing
Imports Syncfusion.Windows.Forms.Tools
Imports System.Drawing
Imports System.Windows.Forms

Namespace TreeViewAdvDemo
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()

			Me.treeViewAdv1.SelectionMode = TreeSelectionMode.MultiSelectAll

			' To ensure that the selected node is highlighted always
			Me.treeViewAdv1.HideSelection = False

			' The appearence of selection rectangle can be changed by following property
			' To identify selected node is highlighted or not when TreeViewAdv loses focus           
			' Set custom colors to the selection rectangle
			Me.treeViewAdv1.InactiveSelectedNodeBackground = New BrushInfo(Color.Green)
			Me.treeViewAdv1.InactiveSelectedNodeForeColor = Color.White
		End Sub
	End Class
End Namespace